### ISSUE TITLE
Use ESLint configuration from commons

### ISSUE THREAD

**Issue by Siegrift** [api3dao/airnode#1897]
See: https://github.com/api3dao/commons/tree/6ebc41c031105fe824132069a4fb80e0415eee78/src/eslint

**Comment by dcroote** [comment]
Isn't this blocked by https://github.com/api3dao/commons/issues/4 ?

**Comment by Siegrift** [comment]
No, it's usable. See: https://github.com/api3dao/commons/tree/main/src/eslint#getting-started and https://github.com/api3dao/signed-api/pull/68

Btw. expect a lot of issues and do not be afraid to silence rules that are too noisy.
